package-lock.json

```diff
@@ -0,0 +1,7042 @@
+{
```

Comment:
the repo uses yarn, package-lock should not be committed

Reply:
Oops. Force of habit. Should be good now.